src/fixed-footer-layout.tsx

```diff
@@ -43,43 +43,48 @@ type Props = {
     onChangeFooterHeight?: (heightInPx: number) => void;
 };
 
-const FixedFooterLayout: React.FC<Props> = ({
+const MIN_AVAILABLE_HEIGHT_FOR_FIXED = 200;
+
+const FixedFooterLayout = ({
     isFooterVisible = true,
     footer,
     footerHeight = 'auto',
     footerBgColor = vars.colors.background,
-    containerBgColor,
+    containerBgColor = vars.colors.background,
     children,
     onChangeFooterHeight,
-}) => {
+}: Props): JSX.Element => {
     const [displayElevation, setDisplayElevation] = React.useState(false);
     const containerRef = React.useRef<HTMLDivElement>(null);
+    const {height: contentHeight} = useBoundingRect(containerRef) || {height: 0};
     const {isTabletOrSmaller} = useScreenSize();
     const {platformOverrides} = useTheme();
     const {height: domFooterHeight, ref} = useElementDimensions();
-    const isWithinIFrame = useIsWithinIFrame();
-    const windowHeight = useWindowHeight();
-    const screenHeight = useScreenHeight();
-    const hasContentEnoughVSpace =
-        windowHeight - domFooterHeight > (isWithinIFrame ? windowHeight : screenHeight) / FOOTER_CANVAS_RATIO;
```

Comment:
I changed this logic. I'm unable to understand the previous one and it doesn't work as expected. With a large content the footer wasn't fixed as expected.

The new implementation is simpler. If the available height (visible height - footer - top distance) is lower than a threshold, the footer is placed below the content, not fixed.

Reply:
I know that @brtbrt was the one who added the previous logic. Maybe he can double check this

Reply:
Given that this component has shown to be very problematic in the past, I'd double check some of the issues linked in previous PRs that modified this code (https://github.com/Telefonica/mistica-web/commits/master/src/fixed-footer-layout.tsx)

Reply:
yep good choice indeed :)

Reply:
I've checked the page you comment in that ticket and it works as expected